fish_prompt.fish

```diff
@@ -644,6 +645,32 @@ function __bobthefish_prompt_k8s_context -S -d 'Show current Kubernetes context'
 end
 
 
+# ==============================
+# Cloud Tools
+# ==============================
+
+function __bobthefish_prompt_aws_vault_profile -S -d 'Show AWS Vault profile'
+    [ "$theme_display_aws_vault_profile" = 'yes' ]
+    or return
+
+    [ -n "$AWS_VAULT" ]
+    or return
+
+    set -l profile $AWS_VAULT
+
+    set -l now (date --utc +%s)
+    set -l expiry (date -d "$AWS_SESSION_EXPIRATION" +%s)
```

Comment:
I guess nothing is guaranteed in software, but according to the doco (and my experience using `aws-vault` and the `aws` CLI in general) this particular env var is written out whenever you execute a profile. See docs [here](https://github.com/99designs/aws-vault#how-it-works).

I was trying hard to not over-complicate it with validation logic because I wasn't sure what the expectation was - but I can totally add some in if you think there's a risk the `date` or `math` will break at all?

Reply:
it probably wouldn't hurt (and wouldn't significantly impact the complexity) to add a truthiness check above, something like...

```fish
    [ -n "$AWS_VAULT" -a -n "$AWS_SESSION_EXPIRATION" ]
     or return
```

Reply:
![image](https://user-images.githubusercontent.com/14284827/92541252-96ac1700-f299-11ea-9dc0-54df3cf61f62.png)

Added check in 5e7bd70; seems to work.